rpa-builder/6.2/modules/ROOT/pages/toolbox-aws-operations-queries-with-aws.adoc

```diff
@@ -0,0 +1,111 @@
+= Queries With AWS
+
+The Queries with AWS Action Step enables you to extract specific data from a document by specifying a query parameter so that the response contains the data you need, without having to apply further filtering on the JSON Output to get the relevant values. 
+
+== Inbound Variables
+
+* *Alias text* 
+* *Query text* 
+* *Directory Path* 
+* *File Name* 
+* *Password*
+* *Read text from page* 
+* *Read text to page*
+* *AWS Credentials*
++
+Or, if *Use custom settings* is activated in the wizard:
++
+** *Client Id*
+** *Client Secret*
+** *Session Token*
+** *Use Session Token*
+** *Region Endpoint*
+* *Json Path*
+
+== Outbound Variables
+
+* *Raw Json*
++
+The complete AWS response.
+* *Normalized Json*
++
+The parsed result based on the normalized result and defined Json path.
+* *First answer* 
++
+A string containing the first result that matched the query. 
+* *All answers* 
++ 
+An array containing all results received from the query. 
+* *Answers count* 
++
+The number of matching results from the specified query. 
```

Comment:
**suggestion**: Maybe it helps to think of the query/result more as a question/answer to better clarify what these outbound parameters represent.